### PR TITLE
monkey patch of werkzeug modules (1.0.0) compatibility

### LINKED ISSUE #196: global: fix import error
Same as:
* [pytest-invenio #42](https://github.com/inveniosoftware/pytest-invenio/issues/42)
* [invenio-oauth2server #196](https://github.com/inveniosoftware/invenio-oauth2server/issues/196)

https://travis-ci.org/inveniosoftware/invenio-oauth2server/jobs/656679343
```
Warning, treated as error:
autodoc: failed to import module u'ext' from module u'invenio_oauth2server'; the following exception was raised:
cannot import name cached_property
The command "./run-tests.sh" exited with 2.
```

Fix also deprecation warning in Travis config:
https://travis-ci.org/inveniosoftware/invenio-oauth2server/jobs/656679343/config

### PR DESCRIPTION
**Pull request by kprzerwa**
closes #196

### CODE REVIEW

File: invenio_oauth2server/_compat.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+"""Fix compatibility with werkzeug due to flask_oauthlib imports."""
```
Comment (slint) [inline invenio_oauth2server/_compat.py]: Don't forget to add the copyright header here

File: setup.py [hunk 2]
```diff
@@ -64,20 +64,19 @@
...
+    'cachelib>=0.1',
     'Flask-Breadcrumbs>=0.4.0',
-    'Flask-Login>=0.3.0',
     'Flask-OAuthlib>=0.9.5',
-    'Flask-WTF>=0.13.1',
-    'Flask>=0.11.1',
+    'Flask-WTF>=0.14.3',
     'future>=0.16.0',
-    'invenio-accounts>=1.0.0',
+    'invenio-accounts>=1.1.3',
```
Comment (slint) [inline setup.py]: I guess once when https://github.com/inveniosoftware/invenio-accounts/pull/303 is merged and v1.2.0 is released, we also bump here

File: setup.py [hunk 3]
```diff
@@ -64,20 +64,19 @@
...
+    'invenio-base>=1.2.2',
+    'invenio-i18n>=1.0.0',
```
Comment (slint) [inline setup.py]:
```suggestion
    'invenio-i18n>=1.2.0',
```

File: tests/helpers.py [hunk 4]
```diff
@@ -10,11 +10,12 @@
 
 from __future__ import absolute_import, print_function
 
-from flask import Blueprint, abort, jsonify, request, session, url_for
-from flask_oauthlib.client import OAuth, prepare_request
-from mock import MagicMock
-from six.moves.urllib.parse import urlparse
-from werkzeug.urls import url_decode, url_parse, url_unparse
+import invenio_oauth2server._compat  # noqa isort:skip
```
Comment (slint) [inline tests/helpers.py]: Hm, is this needed here? (since none of the monkey patching functions are called)

Comment (zzacharo) [reply tests/helpers.py]: This is needed as we saw with @kprzerwa that importing the `_compat` file runs also the code in the `invenio-oauth2server.__init__.py` where the monkeypatching takes place. All in all what we need is just the monkey patch function to be invoked.

Comment (kprzerwa) [reply tests/helpers.py]: trust @zzacharo

**Pull request merged**